angular/projects/spark-angular/src/lib/components/sprk-card/directives/sprk-card-content/sprk-card-content.directive.ts

```diff
@@ -0,0 +1,25 @@
+import { Directive, Input, HostBinding } from '@angular/core';
+
+@Directive({
+  selector: '[sprkCardContent]',
+})
+export class SprkCardContentDirective {
+  /**
+   * The value supplied will be assigned
+   * to the `data-id` attribute on the
+   * element. This is intended to be
+   * used as a selector for automated
+   * tools. This value should be unique
+   * per page.
+   */
+  @HostBinding('attr.data-id')
+  @Input()
+  idString: string;
+
```

Comment:
Some of our directives have an Input for `analyticsString` and others don't. We should figure out whether we want it or not. Then get everything updated one way or the other.

Reply:
@guern1kn  analytics are for clicks. Thats why some have it and some don't. There's no need to add it to elements taht aren't going to have clicks.

Reply:
It doesn't hurt to have on the centered-column. Don't think its worth the breaking change. But, yeah, going forward we'll want to only add it when it makes sense.

Reply:
Sounds like a good plan to me.